plugins/node/opentelemetry-instrumentation-aws-sdk/src/enums.ts

```diff
@@ -15,7 +15,7 @@
  */
 export enum AttributeNames {
   AWS_OPERATION = 'aws.operation',
-  AWS_REGION = 'aws.region',
+  CLOUD_REGION = 'cloud.region',
```

Comment:
`cloud.region` *does* have a constant in `@opentelemetry/semantic-conventions`. However, we can defer using that constant (and the others in this instrumentation) to https://github.com/open-telemetry/opentelemetry-js-contrib/issues/2377

The current constant using the old, deprecated naming (`SEMRESATTRS_CLOUD_REGION`) is/was intended for use as a *Resource* attribute. That was before https://github.com/open-telemetry/semantic-conventions/pull/2238 expanded the use of `cloud.region` beyond resource attributes.  That's why I'm not suggesting this PR use `SEMRESATTRS_CLOUD_REGION`.